markdown/chapters/meandeck/interaction.md

```diff
@@ -412,16 +438,16 @@ a possibility to be considered.
 <row variant="pile">{{!SW}} {{!EoA}} {{!TO}} {{!DD}} {{!DD}}</row>
 
 Here you have a pile where, if they Surgical in response to the cantrip effect
-on Doomsday, and you remove all the copies in the pile, you will have a
+on Doomsday and you remove all the copies in the pile, you will have a
```

Comment:
Opponent can decide not to exile any of the 3 Doomsday copies though 🤔

Reply:
This scenario states you are surgically your own Doomsday....

Reply:
Ah yes I misread that one 👍